pkg/server/fsm.go

```diff
@@ -758,18 +758,31 @@ func capabilitiesFromConfig(pConf *config.Neighbor) []bgp.ParameterCapabilityInt
 	}
 
 	// unnumbered BGP
-	if pConf.Config.NeighborInterface != "" {
-		tuples := []*bgp.CapExtendedNexthopTuple{}
-		families, _ := config.AfiSafis(pConf.AfiSafis).ToRfList()
-		for _, family := range families {
-			if family == bgp.RF_IPv6_UC {
-				continue
-			}
-			tuple := bgp.NewCapExtendedNexthopTuple(family, bgp.AFI_IP6)
-			tuples = append(tuples, tuple)
+	//	if pConf.Config.NeighborInterface != "" {
+	//		tuples := []*bgp.CapExtendedNexthopTuple{}
+	//		families, _ := config.AfiSafis(pConf.AfiSafis).ToRfList()
+	//		for _, family := range families {
+	//			if family == bgp.RF_IPv6_UC {
+	//				continue
+	//			}
+	//			tuple := bgp.NewCapExtendedNexthopTuple(family, bgp.AFI_IP6)
+	//			tuples = append(tuples, tuple)
+	//		}
+	//		caps = append(caps, bgp.NewCapExtendedNexthop(tuples))
+	//	}
+
+	// TODO: it is a hack to enable Extended Next Hop Capability to all enabled Address Families
+	// Extended Nexthop Capability
+	tuples := []*bgp.CapExtendedNexthopTuple{}
```

Comment:
Always advertising `extended nexthop` cap causes any problem on environments that don't need that capability?

Reply:
Not really, the other end if it does not support just will not use it.  I think it is safe to leave as default, at least with cisco bgp implementation it is on by default and not causing any compatibility or operational issues.

Reply:
Great, then simply kill the l.761.

Reply:
done